static/js/rendered_markdown.js

```diff
@@ -190,10 +192,28 @@ exports.update_elements = (content) => {
         $(this).prepend(toggle_button_html);
     });
 
-    // Display the copy-to-clipboard button inside the div.codehilite element.
+    // Display the view-code-in-playground and the copy-to-clipboard button inside the div.codehilite element.
     content.find("div.codehilite").each(function () {
+        const $codehilite = $(this);
```

Comment:
It might be readable to accept an argument to the function `$elt` rather than using `$(this)`.

Reply:
I didn't understand what you meant by this.

Reply:
Rather than using `this`, access the `div.codehilite` element that jQuery passed into this function (which we're currently ignoring).